opentelemetry-sdk/tests/logs/test_log_record.py

```diff
@@ -39,3 +40,10 @@ def test_log_record_to_json(self):
             body="a log line",
         ).to_json()
         self.assertEqual(expected, actual)
+
+    def test_log_record_bounded_attributes(self):
+        attr = {"key": "value"}
+
+        record = LogRecord(timestamp=0, body="a log line", attributes=attr)
+
+        self.assertTrue(isinstance(record.attributes, BoundedAttributes))
```

Comment:
When you make the limits change, can you also add some tests?

Reply:
Added.